include/sot/torque_control/torque-offset-estimator.hh

```diff
@@ -70,7 +70,9 @@ namespace dynamicgraph {
         TorqueOffsetEstimator( const std::string & name );
         void init(const std::string &urdfFile,
                   const Eigen::Matrix4d& _m_torso_X_imu,
-                  const double& gyro_epsilon);
+                  const double& gyro_epsilon,
+                  const int& ffIndex_,
```

Comment:
I think it'd be better to give the joint names rather than the joint IDs as input parameters. Otherwise you force the user to parse the urdf model in python to get the joint indeces from the IDs.